### DISCUSSION TITLE
useVdw seems to be missing from determineBonds in Python wrapper

### DISCUSSION THREAD

**Discussion by filvuk** [rdkit/rdkit#6886] [Q&A] [answered]
The [Python documentation](https://rdkit.org/docs/source/rdkit.Chem.rdDetermineBonds.html#rdkit.Chem.rdDetermineBonds.DetermineBonds) for DetermineBonds and its associated functions states that useVdw can be passed as an optional argument. Unfortunately, when trying to call `rdDetermineBonds.DetermineConnectivity(mol,useVdw=False)`, I get the following error: 

> Python argument types in rdkit.Chem.rdDetermineBonds.DetermineConnectivity(Mol) did not match C++ signature: DetermineConnectivity(RDKit::ROMol {lvalue} mol, bool useHueckel=False, int charge=0, double covFactor=1.3)

Is this a bug or working as intended?

**Comment by dehaenw** [comment, marked as answer]
Check your RDKit version. Works for me on 2023.09.1 Does not work on 2023.03 (as it was a new addition in the latest release!)
```python
import rdkit
rdkit.__version__
```
'2023.09.1'

**Reply by filvuk** [reply]
Well, another reminder to be up to speed on new releases. Thanks for the info!
